source/application/AMPIC887DriverServer/AMGCS2Support.h

```diff
@@ -0,0 +1,125 @@
+#ifndef AMGCS2SUPPORT_H
+#define AMGCS2SUPPORT_H
+
+#include <QChar>
+#include <QString>
+#include "AMGCS2.h"
+class AMGCS2Support
```

Comment:
Is there a reason why you went with using a class rather than a namespace?  Since these are all helper functions it seems like it would be a better fit for a namespace.